rasa/model.py

```diff
@@ -121,18 +121,35 @@ def unpack_model(
     return TempDirectoryPath(working_directory)
 
 
-def get_model_subdirectories(unpacked_model_path: Text) -> Tuple[Text, Text]:
-    """Returns paths for core and nlu model directories.
+def get_model_subdirectories(
```

Comment:
this function needs to be tested / the existing tests for it modified. we should check the new `None` return values and also the new exception